source/_components/sensor.planifneige.markdown

```diff
@@ -20,33 +20,37 @@ In order to use the API, you will need to request an API from the [City of Montr
 
 You will also need to obtain the street side identification codes of the streets you want to monitor. The [street database](http://donnees.ville.montreal.qc.ca/dataset/geobase-double) contains the city's street details, including the `COTE_RUE_ID` which is the street side id required for your configuration.
 
+Once you've obtained an API key, and have identified the streets you want to monitor, you can generate a `configuration.yaml` entry as shown in the below example:
 
-Then add the data to your `configuration.yaml` file as shown in the example:
-
-```yaml
+```
 # Example configuration.yaml entry
-sensor:
-  - platform: planifneige
```

Comment:
Why not `- platform: planifneige`?

Reply:
I had to re-write my code to separate it into a platform component, this is how I figured the config would be best..

Is there a preference for:

```
sensor:
  - platform: planifneige
    REST_OF_CONFIG
```

over:
```
planifneige:
  REST_OF_CONFIG
```

Reply:
I do not really know what the standard is at the moment, it asked more because it struck me.

Reply:
I based myself off of this https://github.com/home-assistant/architecture/issues/131

Reply:
Where is btw the Parent PR in Home Assistant repo?

Reply:
I'm facing some with the tox tests before opening the PR. 
Didn't have enough time to look into it due to work. Should have it done by this weekend.

Reply:
**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** [home-assistant/home-assistant#21343](https://github.com/home-assistant/home-assistant/pull/21343)

I have also updated the PR accordingly.